index.js

```diff
@@ -41,6 +40,11 @@ async function zeroEks (args) {
   if (visualizeCpuProfile) return cpuProfileVisualization(args)
 
   args.title = args.title || `node ${args.argv.join(' ')}`
+
+  if (args.argv && checkForTranspiledCode(args.argv[0])) {
```

Comment:
An empty array also evaluates to `true`. A reliable way to check that `argv` is an array with at least 1 element is:
```js
Array.isArray(args.argv) && args.argv.length > 0
```